database/pom.xml

```
@@ -162,17 +163,65 @@
         </pluginManagement>
       </build>
     </profile>
-    <profile>
+	<profile>
```

```suggestion
    <profile>
```
The formatting looks a bit off.